tests/build/tkn-bundle.go

```diff
@@ -158,8 +158,15 @@ var _ = framework.TknBundleSuiteDescribe("tkn bundle task", Label("build-templat
 				}
 			}
 			bundle := fmt.Sprintf("%s@%s", imgUrl, imgDigest)
-			GinkgoWriter.Print(bundle)
-			fetchImage(fmt.Sprintf("%s@%s", imgUrl, imgDigest), visitor)
+			GinkgoWriter.Printf("Fetching bundle image: %s\n", bundle)
+
+			Eventually(func() error {
+				err = fetchImage(bundle, visitor)
+				if err != nil {
+					return err
+				}
+				return nil
+			}, time.Minute*2, 2*time.Second).Should(Succeed(), "failed to fetch image %q", bundle)
```

Comment:
```suggestion
			Eventually(func() error {
			    fetchImage(bundle, visitor)
			}, time.Minute*2, 2*time.Second).Should(Succeed(), "failed to fetch image %q", bundle)
```

Would this be equivalent?